spec/requests/api/v1/business-intelligence/items_best_day_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe 'Items API' do
+  it 'returns day with most sales' do
+    best_day = '2012-03-22T03:55:09.000Z'
+    other_day = '2012-03-20T23:57:05.000Z'
+
+    item = create(:item)
+
+    invoice_on_best_day = create(:invoice, created_at: best_day)
+    invoice_on_other_day = create(:invoice, created_at: other_day)
+
+    10.times do |n|
+      create(:invoice_item, item: item, invoice: invoice_on_best_day, quantity: 2)
+    end
+
+    4.times do |n|
```

Comment:
If we're not using a parameter in the block, I think we can drop it (that is, lose the `|n|` since we don't use it in the block).  Same for line 13